build/cluster.js

```diff
@@ -0,0 +1,97 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Gulp tasks for kubernetes cluster management.
+ */
+import childProcess from 'child_process';
+import gulp from 'gulp';
+import chmod from 'gulp-chmod';
+import gunzip from 'gulp-gunzip';
+import untar from 'gulp-untar';
+import request from 'request';
+import source from 'vinyl-source-stream';
+
+import conf from './conf';
+
+
+let downScript = `${conf.paths.kubernetes}/cluster/kube-down.sh`,
+    exec = childProcess.exec,
+    file = 'kubernetes.tar.gz',
+    stableVersion = 'v1.0.7',
+    upScript = `${conf.paths.kubernetes}/cluster/kube-up.sh`;
+
+
+/**
+ * @param {function(?Error=)} doneFn
+ */
+function upCluster(doneFn) {
+    exec(`KUBERNETES_PROVIDER=vagrant VAGRANT_DEFAULT_PROVIDER=libvirt ${upScript}`,
+        function (err, stdout) {
+            if (err)
+                return doneFn(new Error(err));
+            console.log(stdout);
+            return doneFn();
+        });
+}
+
+
+/**
+ * Creates cluster from scratch.
+ * Downloads latest version of kubernetes.
+ * Execute script to up cluster.
+ * @param {function(?Error=)} doneFn
```

Comment:
Only functions require `@param` or `@returns` annotation. This statement is just an incovation of gulp.task, which is not a function.